performanceplatform/collector/main.py

```diff
@@ -99,6 +102,9 @@ def _log_collector_instead_of_running(entrypoint, args):
 
 def main():
     args = arguments.parse_args('Performance Platform Collector')
+    if 'collector_slug' in args and args.collector_slug:
```

Comment:
Namespace objects, which is what is being returned by parse_args, don't have a get function.
I'll remove the second clause of the original if statement to make it neater.